### DISCUSSION TITLE
"This day" notification like the OneDrive one

### DISCUSSION THREAD

**Discussion by dada051** [Radiokot/photoprism-android-client#76] [Ideas]
Every day, I receive a notification from the Android OneDrive app, that allow me to see all photos taken this day and month, for all years. It's like a PhotoPrism query "day:07 month:08" (if we are the August 7th). Can be a good feature on this app too.

**Comment by dada051** [comment]
Duplicates https://github.com/Radiokot/photoprism-android-client/discussions/40, don't know why, I searched for "moments", "memories" but not "this day"....

**Reply by Radiokot** [reply]
It's a good idea. Please, upvote #40
